packages/web-twig/src/Resources/components/Button/stories/ButtonColorsSizes.twig

```diff
@@ -8,7 +8,7 @@
       {% for color in colors %}
       <div class="{% if color is same as('inverted') %}docs-Box {% endif %}my-600">
         <div>
-          <Button color={color} size={size} onClick="alert('rimmer')">Button {{ color }}</Button>
+          <Button color={color} size={size} onclick="alert('rimmer')">Button {{ color }}</Button>
```

Comment:
Maybe we should use better alert then `rimmer`...

Reply:
I think I know who is the author :D anyway fixed

Reply:
Thank you, you are the best 👍